app/src/main/java/org/simple/clinic/summary/compose/StatinNudgeView.kt

```diff
@@ -74,6 +79,7 @@ fun StatinNudge(
                 text = stringResource(R.string.statin_alert_at_risk_patient)
             )
           }
+          StainNudgeAddButtons(Modifier.padding(top = 16.dp), statinInfo, addSmokingClick, addBMIClick)
```

Comment:
Can we put these params in separate lines and add param names, please?

Reply:
Done